Source/SourceKittenFramework/String+SourceKitten.swift

```diff
@@ -98,11 +104,17 @@ extension NSString {
     */
     public func NSRangeToByteRange(start start: Int, length: Int) -> NSRange? {
         let string = self as String
-        return string.byteOffsetAtIndex(start).flatMap { stringStart in
-            return string.byteOffsetAtIndex(start + length).map { stringEnd in
-                return NSRange(location: stringStart, length: stringEnd - stringStart)
-            }
+        let startUTF16Index = string.utf16.startIndex.advancedBy(start)
+        let endUTF16Index = startUTF16Index.advancedBy(length)
+        
+        guard let startUTF8Index = startUTF16Index.samePositionIn(string.utf8),
```

Comment:
I don't mean caching the results of this method, but rather storing `utf8` in a variable to reuse it here, rather than calling `string.utf8` twice.